OasisDEX/oasis-borrow
# Hide the stop loss tabs for every Aave product except supported ones (V3 mainnet)

# [Hide the tabs for Aave v2 & Optimism (and arbitrum if that's also set)](https://app.shortcut.com/oazo-apps/story/10241/hide-the-tabs-for-aave-v2-optimism-and-arbitrum-if-that-s-also-set)

  
## Changes 👷‍♀️
- added `supportsAaveStopLoss` function
- used the above function when needed
- changed `AaveProtection` and `AaveProtectionWrite` flags so that it includes V3 (in its name)
  
## How to test 🧪
- try to open Aave V2 position, this should not support automation (no stop loss step before confirming)
- try to open Aave V3 position on mainnet, this should support automation (there is stop loss step before confirming)
- try to open Aave V3 position on optimism, this should not support automation (no stop loss step before confirming)
